var/spack/repos/builtin/packages/py-cmseq/package.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack.package import *
+
+
+class PyCmseq(PythonPackage):
+    """CMSeq is a set of commands to provide an
+       interface to .bam files for coverage and sequence
+       consensus. Used as a dependency for py-metaphlan."""
```

Comment:
```suggestion
       consensus."""
```
This package could be a dependency of many other packages, no need to mention this.